src/exporter/DaemonMetricCollector.cc

```diff
@@ -86,48 +87,60 @@ void DaemonMetricCollector::dump_asok_metrics() {
 
   std::vector<std::pair<std::string, int>> daemon_pids;
 
+  int failures = 0;
   bool sort = g_conf().get_val<bool>("exporter_sort_metrics");
   if (sort) {
-    builder = std::unique_ptr<OrderedMetricsBuilder>(new OrderedMetricsBuilder());
+    builder =
+        std::unique_ptr<OrderedMetricsBuilder>(new OrderedMetricsBuilder());
   } else {
-    builder = std::unique_ptr<UnorderedMetricsBuilder>(new UnorderedMetricsBuilder());
+    builder =
+        std::unique_ptr<UnorderedMetricsBuilder>(new UnorderedMetricsBuilder());
   }
   for (auto &[daemon_name, sock_client] : clients) {
     bool ok;
     sock_client.ping(&ok);
     if (!ok) {
+      failures++;
```

Comment:
I would prefer explicit error messages instead of how many clients were succesful
```suggestion
      dout(10) << "unable to reach " << daemon_name << dendl```

Reply:
That's already there..this is just to show at the end how many were successfully fetched